app/controllers/organen/index.js

```diff
@@ -105,7 +105,7 @@ export default class OrganenIndexController extends Controller {
           .toDate(),
       bindingEinde:
         similarBestuursorgaanInTijd?.bindingEinde &&
-        moment(similarBestuursorgaanInTijd.bindingStart)
+        moment(similarBestuursorgaanInTijd.bindingEinde)
```

Comment:
Should we add a fix for the already created eigen bestuursorganen on prod?

First time tried it but it to the end date of my previous eigen orgaan
![image](https://github.com/user-attachments/assets/cdfccb5e-d379-4fbb-a4dd-5cbf36b09b2a)